bin/improver-spot-extract

```diff
@@ -84,6 +90,24 @@ def main():
         " in altitude to the spot site within the search radius defined when"
         " the neighbour cube was created. May be used with land_constraint.")
 
+    percentile_group = parser.add_argument_group(
+        title="Extract percentiles",
+        description="Extract particular percentiles from probabilistic "
+        "or realization inputs. If deterministic input is provided a warning "
+        "is raised and all leading dimensions are included in the returned "
+        "spot-data cube.")
+    percentile_group.add_argument(
+        "--extract_percentile", default=None, nargs='+', type=int,
+        help="If set to a percentile value or a list of percentile values, "
+        "data corresponding to those percentiles will be returned. For "
+        "example setting '--extract_percentile 50' will result in the "
+        "50th percentile (median) values being returned from a cube of "
+        "probabilities or percentiles.")
```

Comment:
```suggestion
        "probabilities, percentiles or realizations.")
```

Reply:
Modified to give a list example and include realizations. I also added details of the percentile limitation discussed previously (e.g. it doesn't interpolate/resample percentiles at present, so the percentile has to exit within the cube).